### ISSUE TITLE
xDomain not respected for VerticalRectSeries

### ISSUE THREAD

**Issue by sqren** [uber/react-vis#492]
Hi there,

I'm having trouble setting the range for a histogram made with `VerticalRectSeries`. 

```js
  <XYPlot xDomain={[0, 4]} width={500} height={100}>
    <XAxis marginRight={10} />
    <VerticalRectSeries
      data={[{ x0: 0, x: 1, y: 10 }, { x0: 2, x: 3, y: 20 }]}
    />
  </XYPlot>
```

I would expect the above to have x-axis between 0 and 4, but instead the axis goes from -1 to 5.
Should I specify xDomain as `[1,3]` or am I doing something wrong?

Example: https://codepen.io/sqren/pen/weXQmx?editors=0010

**Comment by jckr** [comment]
Great question. Fix incoming 
![image](https://user-images.githubusercontent.com/9727630/27875115-f446736e-6166-11e7-8d0d-97b5d8df1f4f.png)

**Comment by jckr** [comment]
Fixed in 1.6.6

**Comment by sqren** [comment]
Awesome, thanks for the quick resolution!
